homeassistant/components/steam_online/sensor.py

```diff
@@ -170,6 +177,14 @@ def device_state_attributes(self):
         attr = {}
         if self._game is not None:
             attr["game"] = self._game
+        if self._game_id is not None:
+            attr["game_id"] = self._game_id
+            attr[
+                "game_image_header"
+            ] = f"{STEAM_API_URL}{self._game_id}/{STEAM_HEADER_IMAGE_FILE}"
+            attr[
+                "game_image_main"
+            ] = f"{STEAM_API_URL}{self._game_id}/{STEAM_MAIN_IMAGE_FILE}"
```

Comment:
Introduce a local variable for the url including the game id:
```suggestion
            game_url = f"{STEAM_API_URL}{self._game_id}/"
            attr["game_image_header"] = f"{game_url}{STEAM_HEADER_IMAGE_FILE}"
            attr["game_image_main"] = f"{game_url}{STEAM_MAIN_IMAGE_FILE}"
```